packages/portfolio-contract/src/portfolio.flows.ts

```diff
@@ -122,7 +127,7 @@ type AssetMovement = {
 };
 
 const moveStatus = ({ apply: _a, ...data }: AssetMovement) => data;
-const errmsg = (err: any) => ('message' in err ? err.message : `${err}`);
+const errmsg = (err: any) => `${'message' in err ? err.message : err}`;
```

Comment:
While you're here, I'd suggest some protection against nullish err values (which cause `in` to throw).

```suggestion
const errmsg = (err: any) => `${err != null && 'message' in err ? err.message : err}`;
```